ci-operator/step-registry/ipi/conf/openstack/ipi-conf-openstack-ref.yaml

```diff
@@ -6,15 +6,16 @@ ref:
     requests:
       cpu: 10m
       memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: cluster-secrets-openstack-vexxhost
+    mount_path: /output/.config/openstack/vexxhost
+  - namespace: test-credentials
+    name: cluster-secrets-openstack-moc
+    mount_path: /output/.config/openstack/openstack
```

Comment:
"/output" seems to be the home directory of the user executing in the container. 
If this is the case and is always the case, we might be able to get away with mounting a file as
"/output/.config/openstack/clouds.yaml"  and not worry about OS_CONFIG_FILE (I think ${HOME}/.config/openstack is the default location for gophercloud to look in.

Reply:
I don't mind carrying `OS_CONFIG_FILE`. This ensures the CI will keep working even if someone ever decides to change the $HOME to something else.

This also means we don't have to create a new `CLOUD_MOUNT_DIR` variable where we could use `SHARED_DIR`.

Reply:
Harcoded CLOUD_MOUNT_DIR to /var/run/cluster-secret.